tutorials/understanding-group/index.md

```diff
@@ -0,0 +1,327 @@
+
+```json
+{
+    "members": [
+        {
+            "address": "aliceaddr",
+            "weight": "1",
+            "metadata": "president"
+        },
+        {
+            "address": "bobaddr",
+            "weight": "1",
+            "metadata": "treasurer"
+        }
+    ]
+}
+```
+
+
+```sh
+simd tx group create-group $ALICE_KEY "best football association" members.json
+```
+
+
+Let's see the group that we just created:
+
+```sh
+simd query group groups-by-admin $ALICE_KEY
+simd query group group-members 1 # use the group id given by the previous command
+```
+
+Nice! Our group has `best football association` as metadata, Alice as group admin and having Alice and Bob as group members.
+
+## Manage group members
+
+For updating the group members, we send a transaction using the `update-group-members` command.
+We can add a member in our members.json for adding a group member, or set a member voting weight to `0` for deleting the member.
+
+Let's add Carol, Carlos and Charlie as group members and remove Bob:
+
+```json
+{
+    "members": [
+        {
+            "address": "aliceaddr",
+            "weight": "1",
+            "metadata": "president"
+        },
+        {
+            "address": "bobaddr",
+            "weight": "0", // this deletes bob
+            "metadata": "treasurer"
+        },
+        {
+            "address": "caroladdr",
+            "weight": "1",
+            "metadata": "treasurer"
+        },
+        {
+            "address": "carlosaddr",
+            "weight": "1",
+            "metadata": "player"
+        },
+                {
+            "address": "charlieaddr",
+            "weight": "1",
+            "metadata": "player"
+        },
+    ]
+}
+```
+
+```sh
+simd tx group update-group-members $ALICE_KEY 1 members.json
+```
+
+We can verify that the group members are updated:
+
+```sh
+simd query group group-members 1
+```
+
+As an exercice, please re-add bob in the group and go to the next section.
+
+## Create group policy
+
+Next we need to decide of a group policy. This defines how long a proposal can be voted on and how the outcome is calculated.
+Here we use the `ThresholdDecisionPolicy`, it defines a threshold of yes votes (based on a tally of voter weights) that must be achieved in order for a proposal to pass.
+
+Following is the content of the `policy.json`. Proposals can be voted on for a maximum of 10 minutes.
+A proposal requires only one vote to pass.
+
+```json
+{
+    "@type": "/cosmos.group.v1.ThresholdDecisionPolicy",
+    "threshold": "1",
+    "windows": {
+        "voting_period": "10m",
+        "min_execution_period": "0s"
+    }
+}
+```
+
+
+```sh
+simd tx group create-group-policy $ALICE_KEY 1 "" policy.json
+```
+
+Let's verify our newly created group policy and save its address for future use:
+
+```sh
+simd query group group-policies-by-group 1
+export GROUP_POLICY_ADDRESS=$(simd query group group-policies-by-group 1 --output json | jq -r '.group_policies[0].address')
```

Comment:
I wonder if it would be worthwhile to explain why a group policy has a `cosmos123...` address.

Reply:
Good point, added in the terminology section